onnxruntime/core/framework/partial_graph_execution_state.h

```diff
@@ -31,7 +31,7 @@ struct PartialGraphExecutionState {
   ProgramRegion& GetProgramRegions(const SessionState& session_state);
 
   ExecutionContext& GetExecutionContext(gsl::span<const int>& feed_mlvalue_idxs, gsl::span<const OrtValue>& feeds,
-                                        gsl::span<const int>& fetch_mlvalue_idxs, std::vector<OrtValue>& fetches,
+                                        gsl::span<const int>& fetch_mlvalue_idxs, gsl::span<const OrtValue> fetches,
```

Comment:
> gsl::span<const OrtValue> [](http://example.com/codeflow?start=81&length=26)

This will not work. The fetches is an output parameter. Can not replace with gsl::span.